components/dashboard/src/App.tsx

```diff
@@ -71,6 +71,12 @@ function App() {
         }
     }, [localStorage.theme]);
 
+    if (window.location.pathname === '/' && window.location.hash === '' && !hasLoggedInBefore()) {
+        if (window.location.host === "gitpod.io" || window.location.host === "gitpod-staging.com") {
```

Comment:
**nit:** in development the URL is https://at-redirect-www.staging.gitpod-dev.com which won't match this logic.

Reply:
true. in the dashboard component we still don't have a proper solution for the URL mismatch in preview environments.

Reply:
Maybe something like this?

```suggestion
        if (["gitpod.io", "gitpod-staging.com", "gitpod-dev.com", "gitpod-io-dev.com"].some(suffix => window.location.host.endsWith(suffix))) {
```